Wordle.Api/Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class PlayerController(PlayerService playerService) : ControllerBase
+{
+    [HttpGet("TopPlayers")]
+    public async Task<Player[]> GetTopPlayers(int numberOfPlayers = 10)
+    {
+        return await playerService.GetTopPlayers(numberOfPlayers);
+    }
+    
+    [HttpPost("AddPlayer")]
+    public async Task<Player> AddPlayer(Player player)
```

Comment:
I believe the id isn't required and may actually break the post's functionality. I think Entity Framework Core automatically inserts the id based on previous ids (by default it increments the most recent one). So the frontend creating the id doesn't make sense to me - I think EFC should handle it.

Reply:
You are absolutely right. Thanks for suggestion